doc/cookbook/current-route/CurrentRoute.lhs

```diff
@@ -0,0 +1,391 @@
+# How to get current route in servant-server?
+
+This tries to be an answer to the [StackOverflow question](https://stackoverflow.com/questions/41538438/haskell-servant-get-current-route-url-from-handler).
+There are two questions in one:
+
+1. how to get current Request or URL?
+2. how to get current "route"?
+
+Note, that URL (e.g. `/route12/42`) is different than route
```

Comment:
You probably want to remove the ",".

Also, I'd probably add a little something...

> Note that a URL (a string that points to some endpoint or page in your web app, e.g `/route12/42`) is different from a route (the actual endpoint or page, e.g `"route12" :> Capture "id" Int :> Get '[JSON] Int`).

Reply:
yeah, I renamed `/route12`, `/route22` etc to just `/route2`, but not everywhere it seems. Good catch.